### PR TITLE
Add an Experian Check Performed event

### PR DESCRIPTION
**Pull request by VladimirBerdnik**

### CODE REVIEW

File: openapi/components/requestBodies/Webhooks/ExperianCheckPerformed.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+content:
+  application/json:
+    schema:
+      properties:
+        customerId:
+          description: The ID of the checked customer.
+          type: string
+        outcome:
+          description: The check decision where `1` - record matches, `0` - no matches found, `-1` - record mismatches.
```
Comment (XaosSintez) [inline openapi/components/requestBodies/Webhooks/ExperianCheckPerformed.yaml:9]: Does it make sense to make enum here? I also wonder if it makes sense to have string enum here, like `record matches`, `no-matches`. If not - integer enum would be good too?

Comment (VladimirBerdnik) [reply openapi/components/requestBodies/Webhooks/ExperianCheckPerformed.yaml:9]: I decided to keep original Experian value here and not to map it to string representations. Enum should be fine, will do, thanks

**Review by XaosSintez — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

File: openapi/webhooks/experian-check-performed.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+summary: Experian check performed
+operationId: experian-check-performed
+tags:
+  - Customers
```
Comment (adamaltman) [inline openapi/webhooks/experian-check-performed.yaml:4]: Seems like it should be more in the KYC tag?

Comment (VladimirBerdnik) [reply openapi/webhooks/experian-check-performed.yaml:4]: Technically the event is fired on the customer as we have no other entity to link the event with

**Pull request merged**